mora/service/employee.py

```diff
@@ -189,10 +191,10 @@ def get_employee(id):
 
 
 @blueprint.route('/e/<uuid:employee_uuid>/create', methods=['POST'])
-def create_employee(employee_uuid):
+def create_employee_relation(employee_uuid):
     """Creates new employee relations
 
-    .. :quickref: Employee; Create
+    .. :quickref: Employee; Create relation
```

Comment:
What should probably elaborate here — a relation to what? The moon?

Reply:
I tried to be consistent with what is described for the corresponding `/ou/` endpoint.
```
Creates new unit relations

    .. :quickref: Unit; Create relation
```

Reply:
Right, it's fine then :)